src/app/components/Detail/WorkDetailDefinitionList.jsx

```diff
@@ -151,7 +155,7 @@ export const WorkDetailDefinitionList = ({ work }) => {
           (entry, i) => elements.includes(entry[0]) && entry[1] && (
           <React.Fragment key={`"entry"${i.toString()}`}>
             <dt>{workDetailDefinitionLabels[entry[0]]}</dt>
-            <dd>{parseEntries(entry[0], entry[1], workObj)}</dd>
+            { parseEntries(entry[0], entry[1], workObj) && <dd>{parseEntries(entry[0], entry[1], workObj)}</dd> }
```

Comment:
What am I missing here? `parseEntries` always returns an element, so why is the check needed?

Reply:
Aaaahh hmmm I was under the impression they didn't but you're right that they should - this shouldn't get here if parseEntries isn't doing anything.  I'll do a double-check.

Reply:
I think(?) the checks on 155 handle it? It seems like it tests if the field exists on the record and if the value is present before returning.

It certainly doesn't hurt anything, just trying to avoid doubling the number of times that method gets called.